yunwoo/programmers-1/자릿수-더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(n) {
+  let convertedNumber = n
+    .toString()
+    .split("")
+    .map((v) => Number(v));
+
+  return convertedString.reduce((acc, cur) => acc + cur);
+}
```

Comment:
```suggestion
function solution(n) {
  return n
    .toString()
    .split("")
    .reduce((acc, cur) => acc + Number(cur), 0);
}
```
- 잘푸셨네요 고생하셨습니다 👍 map을 돌리고 reduce를 하기보다 바로 reduce를 활용해서 결과를 도출하는 방법이 더 효율적인 방법인 것 같습니다 !!

Reply:
헛 reduce안에서 변환해줄 생각을 못했네요..! 리뷰감사합니다🙇‍♂️